### ISSUE TITLE
card-tools not working with HA 0.88.1

### ISSUE THREAD

**Issue by derandiunddasbo** [thomasloven/lovelace-card-tools#14]
Upgrading to HA 0.88.1 seems to break card-tools and all plugins depending on it. These are the errors from the javascript console:

```
common.js:206 Logging system initialised at Mon Feb 25 2019 16:55:31 GMT+0100 (Mitteleuropäische Normalzeit)
card-tools.js?v=297fcc7:230 CARD-TOOLS IS INSTALLEDDeviceID: xxx-yyy
card-tools.js?v=297fcc7:15 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.getPrototypeOf (<anonymous>)
    at Function.litElement (card-tools.js?v=297fcc7:15)
    at customElements.whenDefined.then (fold-entity-row.js?v=6d11d3c:2)
litElement @ card-tools.js?v=297fcc7:15
customElements.whenDefined.then @ fold-entity-row.js?v=6d11d3c:2
Promise.then (async)
(anonymous) @ fold-entity-row.js?v=6d11d3c:1
card-tools.js?v=297fcc7:15 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.getPrototypeOf (<anonymous>)
    at Function.litElement (card-tools.js?v=297fcc7:15)
    at customElements.whenDefined.then (auto-entities.js?v=be13234:2)
litElement @ card-tools.js?v=297fcc7:15
customElements.whenDefined.then @ auto-entities.js?v=be13234:2
Promise.then (async)
(anonymous) @ auto-entities.js?v=be13234:1
```

The plugins are working again, when downgrading to 0.87.1.

**Comment by MatteGary** [comment]
I get this error

`https://xxxx.xxxx.com/local/lovelace-ui-addon/card-tools.js?v=0.0.3:1:16 Uncaught NotSupportedError: Failed to execute 'define' on 'CustomElementRegistry': this name has already been used with this registry`

**Comment by thomasloven** [comment]
@derandiunddasbo The version of card tools indicated in your error was **a month** old at the time of posting.

For future reference, the first thing you do when something doesn't work after upgrading (or really ever) is to get the lastest version of everything involved.

**Comment by derandiunddasbo** [comment]
Thanks for the hint, the latest version solved the issue.

> For future reference, the first thing you do when something doesn't work after upgrading (or really ever) is to get the lastest version of everything involved.

Don't get me wrong, in my job I have to write these "...did you check with the latest version..." messages every now and then myself to our users, but I usually try to assume the user _inadvertently_ missed the update before reporting the issue. Users tend to have at least as much bad days as developers. ;)

**Comment by thomasloven** [comment]
Great! Glad to hear it!

It was meant as a handy tip for the future - in case you missed the debugging hints mentioned in the install guide linked from the readme.
Things break all the time, in Home Assistant or in my stuff, and keeping things backwards compatible is really tough, so I fully understand that it can be hard for the average user to keep up - hence the reminder.
But I seriously lost count on how many times I typed "Update card-tools" after the release of Home Assistant version 0.88, and that's with the fix actually being released a week *before* 0.88.0 ...

I meant no offense, and I'm sorry if you took it the wrong way.

**Comment by derandiunddasbo** [comment]
> But I seriously lost count on how many times I typed "Update card-tools" after the release of Home Assistant version 0.88, and that's with the fix actually being released a week before 0.88.0 ...

Yes, I can imagine that. My personal experience is, that some bold or italic text right below the download button with a kind reminder to first update to the latest version(s) first before reporting any issues is actually helping to reduce unnecessary tickets. Doesn't prevent _all_ of them, but raises awareness significantly.

> I meant no offense, and I'm sorry if you took it the wrong way.

No offense taken. :-)
